neural_compressor/adaptor/ox_utils/weight_only.py

```diff
@@ -294,6 +306,7 @@ def rtn_quantize(
         group_size (int, optional): how many elements share one scale/zp. Default is 32.
         scheme (str, optional): sym or asym. Defaults to "asym".
         ratios (dict, optional): percentile of clip. Defaults to {}.
+        accuracy_level (int): accuracy level
```

Comment:
Suggest writing in more detail, including the acceptable values and the meanings of the different values